src/_includes/docs/install/reqs/macos/zsh-config.md

```diff
@@ -0,0 +1,51 @@
+{% assign target = include.target %}
+
+<details markdown="1">
+<summary><strong>To verify your shell configuration, expand this section</strong></summary>
+
+Like most UNIX-like operating system, macOS can support multiple shells,
+like `bash`, `zsh`, and `sh`.
+
+As of the October 2019 release of macOS Catalina (macOS 10.15),
+Z shell or `zsh` is the default shell for macOS.
+Apple made this change to due to licensing changes for `bash`.
```

Comment:
Take this line out, we don't need it.
```suggestion
```

Reply:
Can do. I waivered on this, though, because the _why_ of it kept creeping into my mind.